### PR TITLE
Fix occasional package problems related to dependencies. 

### PR DESCRIPTION
**Pull request by jmchilton**
(Please replace this header with a description of your pull request. Please include *BOTH* what you did and why you made the changes. The "why" may simply be citing a relevant Galaxy issue.)
(If fixing a bug, please add any relevant error or traceback)
(For UI components, it is recommended to include screenshots or screencasts)

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: packages/app/setup.cfg [hunk 1]
```diff
@@ -50,6 +50,7 @@ install_requires =
     celery
     cloudauthz==0.6.0
     gxformat2
+    importlib_metadata<5  # Work around kombu bug
```
Comment (nsoranzo) [inline packages/app/setup.cfg]:
```suggestion
    importlib-metadata<5  # Work around https://github.com/celery/kombu/issues/1600
```

File: pyproject.toml [hunk 2]
```diff
@@ -60,6 +60,7 @@ gravity = ">=1.0"
 gunicorn = "*"
 gxformat2 = "*"
 h5py = "*"
+importlib_metadata = "<5"  # Work around kombu bug
```
Comment (nsoranzo) [inline pyproject.toml]:
```suggestion
importlib-metadata = "<5"  # Work around https://github.com/celery/kombu/issues/1600
```

File: pyproject.toml [hunk 3]
```diff
@@ -60,6 +60,7 @@ gravity = ">=1.0"
...
 importlib_resources = "*"
```
Comment (nsoranzo) [inline pyproject.toml]:
```suggestion
importlib-resources = "*"
```

**Review by nsoranzo — APPROVED** [review]

**Comment by nsoranzo** [comment]
Target 23.0?

**Comment by jmchilton** [comment]
It looks like 22.05 has other problems - https://github.com/galaxyproject/galaxy/actions/runs/4384709748/jobs/7676508721 - I'll open a PR for 23.0 though.

**Comment by martenson** [comment]
I like how the backport made it to the codebase faster than the initial :D

**Pull request merged**